images/scripts/arch.setup

```diff
@@ -106,6 +106,33 @@ echo "%wheel ALL=(ALL) ALL" > /etc/sudoers.d/90-cockpit-wheel
 useradd -c Builder builder
 echo "builder ALL = NOPASSWD: /usr/sbin/makechrootpkg, /usr/sbin/mkarchroot, /usr/bin/*-x86_64-build" > /etc/sudoers.d/90-archbuild
 
+BUILD_DEPS="\
+accountsservice \
+base-devel \
+cockpit \
+git \
+glib-networking \
+gobject-introspection \
+gtk-doc \
+intltool \
+json-glib \
+krb5 \
+libgsystem \
+libssh \
+libvirt-dbus \
+networkmanager \
+npm \
+pcp \
+perl-json \
+perl-locale-po \
+xmlto \
+"
+
+# Create build chroot for cockpit projects for offline builds, the base chroot
+# is created in /var/tmp/archbuild/root
+mkdir /var/tmp/archbuild
```

Comment:
Nitpick, but /var/tmp/ is prone to getting cleaned up automatically. Better use /var/lib/ or /var/cache/.